docs/server/settings.rst

```diff
@@ -250,6 +250,32 @@ Configuration settings for applications used by Pootle.
   - ``label`` specifying the text that will be displayed next to the mark.
 
 
+.. setting:: POOTLE_SCORE_COEFFICIENTS
+
+``POOTLE_SCORE_COEFFICIENTS``
+  Default::
+
+    {
+        'EDIT': 5.0/7,
+        'REVIEW': 2.0/7,
+        'SUGG': 0.2,
```

Comment:
@julen suggested to use `SUGGEST` in https://github.com/translate/pootle/pull/4427#issuecomment-178577366